import-service/src/handlers/get-import-products-file.ts

```diff
@@ -0,0 +1,57 @@
+import S3 from 'aws-sdk/clients/s3';
+import { APIGatewayProxyHandler } from 'aws-lambda';
+
+const {
+  S3_BUCKET_NAME,
+  S3_BUCKET_FOLDER_UPLOADED,
+  S3_BUCKET_CONTENT_TYPE,
+  S3_BUCKET_REGION,
+  S3_BUCKET_SIGNED_URL_EXPIRES,
+  SECRET_ACCESS_KEY,
+  ACCESS_KEY_ID
+} = process.env;
+const DEFAULT_HEADERS = {
+  'Access-Control-Allow-Origin': '*'
+};
+
+const getImportProductsFile: APIGatewayProxyHandler = async event =>  {
+  console.log('Event data: ', event);
+  let body;
+  let statusCode = 200;
```

Comment:
Seems like by REST better to use 202 (Accepted) status code here.

Reply:
Disagree :)

This service is responsible for URL creation to PUT file later, it's job is done at the end and the payload is the URL.

200 OK means that the request has succeeded and the processing of our request is done. The response is the final payload and the service will not take further actions.

202 Accepted on the other hand means that the request have been accepted for processing, and the service will now start. This does not mean that the processing itself is a success, nor that the process have ended.